cli/tests/integrations/test_help.py

```diff
@@ -34,6 +34,7 @@ def test_list():
 \tpackage        \tInstall and manage DCOS software packages
 \tservice        \tManage DCOS services
 \ttask           \tManage DCOS tasks
+\ttunnel         \tRun a tunnel through DCOS cluster nodes
```

Comment:
@nlsun what is the rationale for having this be its own subcommand instead of a part of the 'node' subcommand?